charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -344,6 +344,10 @@ spec:
             value: {{ .Values.datadog.prometheusScrape.version | quote }}
           {{- end }}
           {{- end }}
+          {{- if (((.Values.datadog.autoscaling).workload).enabled) }}
```

Comment:
should we also check remote-config enabled option?

Reply:
I have added a check in NOTES